AdaptiveCards/templating/language.md

```diff
@@ -53,21 +61,21 @@ You can bind to the data within the `body` or `actions` of the card.
     "body": [
         {
             "type": "TextBlock",
-            "text": "Hi {employee.name}! Here's a bit about your org..."
+            "text": "Hi ${employee.name}! Here's a bit about your org..."
         },
         {
             "type": "TextBlock",
-            "text": "Your manager is: {employee.manager.name}"
+            "text": "Your manager is: ${employee.manager.name}"
         },
         {
             "type": "TextBlock",
-            "text": "3 of your peers are: {employee.peers[0].name}, {employee.peers[1].name}, {employee.peers[2].name}"
+            "text": "3 of your peers are: ${employee.peers[0].name}, ${employee.peers[1].name}, ${employee.peers[2].name}"
         }
     ]
 }
 ```
 
-## Separating the template from the data
+### Option B: Separating the template from the data
 
 Alternatively (and more likely), you will create a re-usable card "template" without including the data. This template could be stored as a file and added to source control.
```

Comment:
Reformed using [contractions](https://styleguides.azurewebsites.net/Styleguide/Read?id=2700&topicid=28361) ("you'll" instead of "you will")
```suggestion
Alternatively (and more likely), you'll create a re-usable card template without including the data. This template could be stored as a file and added to source control.
```